Alignment/APEEstimation/test/SkimProducer/skimProducer_cfg.py

```diff
@@ -178,13 +182,13 @@
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
 from Configuration.AlCa.GlobalTag import GlobalTag
 
-if isData:
-    process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_data', '')
-    #process.GlobalTag = GlobalTag(process.GlobalTag, '101X_dataRun2_Prompt_v11', '')
-elif isMc:
-    #process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:phase1_2018_realistic', '')
-    process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:phase1_2018_design', '')
-
+if globalTag == None:
+    if isData:
+        process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_data', '')
+    elif isMc:
+        process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_mc', '')
```

Comment:
Is `auto:run2_mc` ("post-VFP" 2016 MC) a sensible default? It seems that for MC it doesn't make sense to specify a default global tag, and should always be specified explicitly.

Reply:
Indeed, the global tag to use changes per skim. This option is there to allow changing the global tag centrally and was kept from previous versions of the tool. auto:run2_mc is used here only because it was the last used option for a post-VFP APE measurement. This part of the code is probably not necessary anymore.

Reply:
@christopheralanwest I removed that part from the code now.